kubecolor.yml

```diff
@@ -0,0 +1,21 @@
+theme:
+  get:
+    header: red
+  describe:
+    header: blue
+  all:
```

Comment:
Probably wise to split up the "all" and per sub-command settings, just in case someone has a kubectl plugin named "all"

Such as:

```yaml
theme:
  header: green

overrides:
  get:
    header: red
```

I wonder too if we would like to get flag parsing in there? Or at least leave room for a future implementation to add that? Such as:

```yaml
theme:
  header: green

overrides:
  - command: get pods
    flags: --namespace=? --all-namespaces
    theme:
      header: red
```

Not my best design proposal, tbh. But maybe something along those lines?

Reply:
I like the first proposal...  On the 2nd one I'm wondering on the implementation as it feels like infinite possibilities (what about `-o wide` or `-o yaml` flags ? What about how each plugins output format is ? ex: `k resource-capacity --pods --util` 
)

Reply:
I mean, we need to rework `printer/kubectl_output_colored_printer.go` a lot to make it more open to other commands

Reply:
The overrides could also define what kind of printer to use. For example to use the table printer for certain combinations of args and flags, and the describe printer for certain other, etc.